### DISCUSSION TITLE
Wix 4.0 Error WIX0200 The Package element contains an unhandled extension element 'WixUI'

### DISCUSSION THREAD

**Discussion by ijuedt** [wixtoolset/issues#8139] [Questions] [answered]
- Im using latest Visual Studio 2019 (16.11.35), have installed HeatWave extension (V1.0.4), also tried fresh re-install after last VS update.
- Project file shows `Sdk="WixToolset.Sdk/4.0.2"`
- Installed extensions are WixToolset.Netfx.wixext, WixToolset.UI.wixext, and WixToolset.Util.wixext in version 5.0.0.
- My xmlns includes are
  - `xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui"`
  - `xmlns:NetFX="http://wixtoolset.org/schemas/v4/wxs/netfx"`
  - `xmlns:util="http://wixtoolset.org/schemas/v4/wxs/util"`
- wxs file has been converted from wix 3.
- My UI Element is `<ui:WixUI Id="WixUI_FeatureTree" />`

Might this be a version mismatch between wix 4 and wix 5?

**Comment by ijuedt** [comment, marked as answer]
Found out myself: there were additional errors unrelated to wix toolset which were kind of hiding the real problem, while my UI-related error hid those other errors. After temporarily excluding my UI element and removing the errors, the real error was revealed: my extensions did "not have support for WiX Toolset v4.0.2". Then I only had to change the version declaration to "5.0.0".

**Comment by robmen** [comment]
I think we're leaving issues open so they are easier for others to find.
